pkg/tls/tls.go

```diff
@@ -47,10 +46,18 @@ func GetCertificatesFromSecret(client kclient.Client, secretName, secretNamespac
 	return secret.Data, nil
 }
 
-const (
-	// TLSCertKeySuffix is the key suffix for tls certificates in a kubernetes secret.
-	TLSCertKeySuffix = ".crt"
-)
+// isCertificateProvided returns true if the name and the namespace of the secret which holds
+// the certificate were provided. Otherwise it returns false.
+func isCertificateProvided(cerificateName, certificateNamespace string) (bool, error) {
+	if cerificateName == "" && certificateNamespace == "" {
+		return false, nil
+	}
+	if cerificateName == "" || certificateNamespace == "" {
```

Comment:
Theoretically we can assume that the certificate is in the same namespace, so the namespace can be nil, can we ? Otherwise, we have to explicitly specify that both entries should be provided.

Reply:
The PR was changed to mount the secret so the namespace is not specified in values.yaml